src/GridFieldEditableColumns.php

```diff
@@ -97,7 +97,6 @@ public function getColumnContent($grid, $record, $col)
         $value = $this->formatValue($grid, $record, $col, $value);
```

Comment:
Am I right in thinking that now this line and the previous assignments of `$value` are obsolete?

Reply:
Correct, `$value` isn't used in the rest of the function and should be removed